src/portfft/committed_descriptor.hpp

```diff
@@ -0,0 +1,315 @@
+/***************************************************************************
```

Comment:
In my opinion for a header only library everything should be in the `include` folder, and `src` should not even be there.

Reply:
I'd say it can make sense to have both. Note that we have a similar structure for portBLAS and portDNN. These libraries can be used as a shared library too but the headers that are actually implementing the features are in `src`. I'm curious to hear other's opinions too

Reply:
We'll move everything to `include` in a separate PR.